nengo/spa/assoc_mem.py

```diff
@@ -100,18 +95,22 @@ def __init__(self, input_vocab, output_vocab=None,  # noqa: C901
             if wta_output:
                 self.am.add_wta_network(wta_inhibit_scale, wta_synapse)
 
-            if threshold_output:
-                self.am.add_threshold_to_outputs()
+            if cleanup_output:
+                self.am.add_cleanup_output(
+                    replace_output=replace_output_with_cleaned_output)
 
             self.input = self.am.input
             self.output = self.am.output
 
+            if cleanup_output and not replace_output_with_cleaned_output:
+                self.cleaned_output = self.am.cleaned_output
+
             if inhibitable:
                 self.inhibit = self.am.inhibit
 
-            self.utilities = self.am.utilities
-            if threshold_output:
-                self.thresholded_utilities = self.am.thresholded_utilities
+            self.utilities = self.am.output_utilities
```

Comment:
Ah. It's a side effect of how the new networks associative memory code works. The networks code now creates a utilities node for each output, the default output that is created in the constructor makes a default `output_utilities` node as well (it's just the `output name` + '_utilities'). I kept it as `utilities` in the spa code so that minimal spa code changes are needed.